news/9722.feature.rst

```diff
@@ -0,0 +1 @@
+OpenVMS support. Logical name translation by mask is done by DCL command.
```

Comment:
I’m assuming this would make sense for OpenVMS users, and unintelligible to me due to myself having basically zero knowledge about OpenVMS.